airflow/opt/providers/etna/etna/etls/linkers/single_cell_rna_seq.py

```diff
@@ -0,0 +1,112 @@
+"""
+Module container helper functions for processing metis files associated with both raw and processed single cell data,
+both in pools and non pools.  Notably, these functions are versioned to account for changes in the pipeline that may not
+necessaril ybe backported to all data sets.
+
+Most of these helpers are accessible on the main MetisEtlHelper class as methods.
+"""
+from typing import Mapping, Optional, List, Tuple
+
+import re
+
+from airflow.exceptions import AirflowException
+from airflow.models.xcom_arg import XComArg
+from airflow.utils.task_group import TaskGroup
+from etna.etls.metis import MatchedRecordFolder
+
+from etna import MetisEtlHelpers, link
+
+timepoint_regex = re.compile(r"^(?P<timepoint>[A-Z0-9]+-[A-Z0-9]+-[A-Z][A-Z]?\d+)[A-Z0-9]+.*")
```

Comment:
@arkal 
Generalized these regexes so they can attempt to process even specialized identifiers.

Note below that I throw and fail all processing though if a record folder does not even match these wide regexes, as that would generally imply incorrect naming of something.